clang/lib/Driver/Driver.cpp

```diff
@@ -6553,6 +6635,19 @@ const ToolChain &Driver::getOffloadingDeviceToolChain(
                                                            HostTC, Args);
       break;
     }
+    case Action::OFK_SYCL:
```

Comment:
May be a TODO here that additional targets will be added?

Reply:
Addressed in recent commit.